docs/source/getting_started/installation.rst

```diff
@@ -113,17 +99,16 @@ character, this shows the start of your terminal prompt).
 
```

Comment:
Explain what the results of these commands should be?

Reply:
It might be a pain as it always installs the latest version, so the expected output will keep changing.

Reply:
Use an 'at least' value rather than a precise one. My Docker is 18.09.2 so I'd put make sure it's 18 or later. That sort of thing